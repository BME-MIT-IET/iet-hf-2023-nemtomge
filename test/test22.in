loadmap map.json 
start V1 V2 
set Scientist V1 PL1 
set Scientist V2 PL1 
give Crafted V1 Stun 3 
show Active V2 
use V1 V2
1 
show Active V2
exit
