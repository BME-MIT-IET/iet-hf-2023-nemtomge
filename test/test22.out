Map from file map.json loaded
The game has been started. Players are: V1 V2 
Scientist V1 set on field PL1 
Scientist V2 set on field PL1 
Scientist V1 has been given crafted agent Stun time: 3 
V2 doesn't have any active effects
Choose one to use: 
0 - cancel
1 - Stun(3)
V2's active effects are:
Stun(3)
